index.html

```diff
@@ -217,8 +217,6 @@ <h1 id="ovningar">Övningar</h1>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
     <script src="vendor/highlight.pack.js"></script>
     <script src="vendor/bootstrap.min.js"></script>
-    <script src="vendor/smooth-scroll.js"></script>
```

Comment:
> What do you mean? This smooth scrolling is about attaching it to anchor links on the same page, instead of having the page jump around when clicking the nav links in the menu. Great UX, if you ask me.

Attaching what to anchor links? I mean, the event listener is on all ["data-scroll"](https://github.com/introprogramming/introprogramming.github.io/blob/master/vendor/smooth-scroll.js#L207) - I've no idea what that is but it's not only anchors. The browser scrolling is disabled [here](https://github.com/introprogramming/introprogramming.github.io/blob/master/vendor/smooth-scroll.js#L149). If something is buggy beyond that, there is no fallback.

Anyways, the biggest reason I'm concerned is that this page is already such a spaghetti dish that I would love to see it given a proper shaving.